packages/api/src/schema/api-schema.yaml

```
@@ -1054,6 +1104,7 @@ components:
         canPublish:
           type: boolean
           description: Whether a user is allowed to publish audio/video tracks
+          example: true
```

Does this one default to true as well? A `default` flag might be more useful than an `example` in these cases where the default behavior is not the "zero" value for the type.
```suggestion
          default: true
```

---

Makes sense, the auto generated docs seem to need the example field though, so should I set both? I'll do that for now, but let me know.

---

Ah, yeah, do that. My comment was based on existing OpenAPI tools that we've used in the past like redoc, but I'm actually not sure what our current implementation is doing.